packages/create-pantheon-decoupled-kit/src/templates/next-drupal-search-api-addon/components/search-input.jsx

```diff
@@ -9,30 +9,35 @@ const SearchInput = () => {
 	const onSearch = (event) => {
 		event.preventDefault();
 
-		if (typeof searchQuery !== 'string') {
+		if (!searchQuery) {
 			return;
 		}
 
-		const encodedSearchQuery = encodeURI(searchQuery);
-		router.push(`/search/${encodedSearchQuery}`);
+		const encodedSearchQuery = encodeURI(searchQuery);	
```

Comment:
```suggestion
		const encodedSearchQuery = encodeURI(searchQuery);
```
looks like some extra space was added?